multinet/user.py

```diff
@@ -137,3 +138,20 @@ def filtered_user(user: User) -> FilteredUser:
 def copy_user(user: User) -> User:
     """Create and return a new instance of User."""
     return from_dict(User, dataclasses.asdict(user))
+
+
+def search_user(query: str) -> Cursor:
+    """Search for users given a partial string."""
+
+    coll = user_collection()
+    aql = read_only_db("_system").aql
+
+    query = f"""
+        FOR doc in {coll.name}
+          FILTER CONTAINS(LOWER(doc.name), LOWER('{query}'))
+            OR CONTAINS(LOWER(doc.email), LOWER('{query}'))
+
+          RETURN doc
```

Comment:
Returning just the doc without a limit might be an issue. If you just search for `gmail`, you'll get a ton of users, which might crush the server (once we have enough users)

Reply:
Agreed. Putting in a limit of 50 should be good for now.